src/components/calculator/PrevalenceControls.tsx

```diff
@@ -1,9 +1,12 @@
+import countries from 'i18n-iso-countries'
 import { isNumber } from 'lodash'
 import React, { useEffect, useState } from 'react'
 import { Card, Form, InputGroup } from 'react-bootstrap'
 import { Typeahead } from 'react-bootstrap-typeahead'
 import { Trans, useTranslation } from 'react-i18next'
 
+import i18n from '../../i18n'
```

Comment:
I think we've been trying to not do relative imports. Could you make this an absolute import instead?

Reply:
Sure, done.